great_expectations/rule_based_profiler/data_assistant_result/plot_result.py

```diff
@@ -24,3 +24,6 @@ class PlotResult:
     """
 
     charts: List[alt.Chart]
+
+    def __repr__(self):
+        return ""
```

Comment:
Could we annotate with `-> str`. Also, is this the recommend way to remove a repr?

Reply:
I updated the annotation! ✅

Reply:
As far as I can find this is the preferred method, but please let me know if you see something else!